src/legacy/ui/public/courier/courier.js

```diff
@@ -21,12 +21,12 @@ import _ from 'lodash';
 
 import { timefilter } from 'ui/timefilter';
 
-import '../es';
-import '../directives/listen';
-import '../index_patterns';
-import { uiModules } from '../modules';
-import { addFatalErrorCallback } from '../notify';
-import '../promises';
+import 'ui/es';
```

Comment:
Why did you have to go ahead and change the imports?
I actually think there was a PR a few months back taking these imports the opposite direction (ui -> relative).

Reply:
I didn't have to; I wanted to change `ui/index_patterns` to make it easier to migrate later. So I changed the rest for consistency. There was a PR converting these to relative imports about a year and a half ago, but nothing more recent. 

As I'm looking at this again, I don't know why this implicit import is even here to begin with; looks like it is leftover from a long time ago (Kibana 4.x). So I'll just revert all of the changes here and remove the reference to `index_patterns` entirely.